Mock node-postal to allow testing without libpostal loaded

orangejulius opened the pull request:
Add a mock libpostal library that has a hardcoded list of libpostal responses. This allows us to ensure libpostal does not have to be loaded, both to speed up the tests in general and to allow the tests to run in environments without libpostal installed, namely Travis.

orangejulius commented:
Yeah, as it turns out I'm going to have to completely rewrite this, since it turns out we actually call `node` directly with `child_process.spawnSync`, so proxyquire won't help us. The code itself will have to look at an environment variable and decide whether to load the real libpostal or the mock.

missinglink commented:
tests are now passing on Travis, I used the `ppa:ubuntugis/ppa` to install a more modern version of `ogr2ogr`.

orangejulius commented:
I've rebased this to fix the conflicts, updated npm-check in my branch. This is now ready for review and once reviewed can be merged! After that we can update https://github.com/pelias/interpolation/pull/60 to work with the libpostal mock library.
